webpack-dll.config.js

```diff
@@ -0,0 +1,62 @@
+/**
+ * External Dependencies
+ */
+const path = require( 'path' );
+const webpack = require( 'webpack' );
+
+/**
+ * Internal Dependencies
+ */
+const config = require( './server/config' );
+
+const bundleEnv = config( 'env' );
+
+module.exports = {
+	entry: {
+		vendor: [ path.join( __dirname, 'client', 'vendor-dll.js' ) ]
```

Comment:
inlined it in https://github.com/Automattic/wp-calypso/pull/8161/commits/b544c2fa0e396b2ae95e81ac4653602abf34dd12